### PR TITLE
v4: e2e: Don't manually "docker kill"

### PR DESCRIPTION
**Pull request by thockin**
It happens automatically on test cleanup

### CODE REVIEW

File: test_e2e.sh [hunk 1]
```diff
@@ -1066,12 +1066,12 @@ function e2e::askpass_url() {
         --root="$ROOT" \
         --link="link" \
         >> "$1" 2>&1 || true
-    docker_kill "$CTR"
 
     # check for failure
     assert_file_absent "$ROOT"/link/file
 
     # run with askpass_url service with correct password
+    docker_kill "$CTR"
```
Comment (stp-ip) [inline test_e2e.sh:1074]: Why the manual kill here? To force a kill before CTR gets rewritten?

Comment (thockin) [reply test_e2e.sh:1074]: correct - CTR is a helped and I need to kill and restart it here

**Comment by thockin** [comment]
Rebased

**Review by stp-ip — APPROVED** [review]
/lgtm

**Pull request merged**